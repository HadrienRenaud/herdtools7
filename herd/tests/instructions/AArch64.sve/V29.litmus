AArch64 V29
(* Output vector lenght, in bytes *)
Variant=sve:512
{
uint64_t 0:X0;
}
    P0         ;
  CNTB X0      ;
forall 0:X0=64 ;
