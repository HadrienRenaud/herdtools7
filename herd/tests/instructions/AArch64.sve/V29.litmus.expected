Test V29 Required
States 1
0:X0=16;
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (0:X0=16)
Observation V29 Always 1 0
Hash=c0179047f87d6ae87f6e7c40bee8fd71

